### PR TITLE
Add additional perf cluster configs

### PR DESCRIPTION
**Pull request by adamcfraser**
#### Fixes #.

- [x] Ran `flake8`

#### Changes proposed in this pull request:

Adds cluster configs for running with 6 CBS nodes in several more sg/ac combinations, and adds 16/4 sg/ac.

### CODE REVIEW

File: libraries/utilities/generate_clusters_from_pool.py [hunk 1]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
...
         ClusterDef("8sg_4ac_3cbs_8lgs", num_sgs=8, num_acs=4, num_cbs=3, num_lgs=8, num_lbs=0),
+        ClusterDef("8sg_4ac_6cbs_8lgs", num_sgs=8, num_acs=4, num_cbs=3, num_lgs=8, num_lbs=0),
         ClusterDef("8sg_8ac_3cbs_8lgs", num_sgs=8, num_acs=8, num_cbs=3, num_lgs=8, num_lbs=0),
+        ClusterDef("8sg_8ac_6cbs_8lgs", num_sgs=8, num_acs=8, num_cbs=3, num_lgs=8, num_lbs=0),
         ClusterDef("8sg_12ac_3cbs_8lgs", num_sgs=8, num_acs=12, num_cbs=3, num_lgs=8, num_lbs=0),
         # 16 sync_gateways
+        ClusterDef("16sg_4ac_3cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
+        ClusterDef("16sg_4ac_6cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
         ClusterDef("16sg_8ac_3cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
+        ClusterDef("16sg_8ac_6cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
```
Comment (raghusarangapani) [inline libraries/utilities/generate_clusters_from_pool.py]: test_generate_clusters_from_pool.py will need update as well

**Comment by adamcfraser** [comment]
@raghusarangapani What's that test validating?  I can't tell what 
https://github.com/couchbaselabs/mobile-testkit/blob/master/mobile_testkit_tests/test_generate_clusters_from_pool.py#L95
is intended to test.

**Comment by raghusarangapani** [comment]
This is a unit test for the cluster config generator generate_clusters_from_pool.py. It verifies that the right number of files are generated. If ip_to_node_type is defined, it will also verify that the right IP are set to be servers, accels etc. Since this change is adding more configs, if you just run the test, it will assert an tell you that the number of files are changed. You might have to just update that number.

File: libraries/utilities/generate_clusters_from_pool.py [hunk 2]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
         ClusterDef("2sg_8ac_6cbs_2lgs", num_sgs=2, num_acs=8, num_cbs=6, num_lgs=2, num_lbs=0),
         # 4 sync_gateways
         ClusterDef("4sg_2ac_3cbs_4lgs", num_sgs=4, num_acs=2, num_cbs=3, num_lgs=4, num_lbs=0),
+        ClusterDef("4sg_2ac_6cbs_4lgs", num_sgs=4, num_acs=2, num_cbs=3, num_lgs=4, num_lbs=0),
```
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py [hunk 3]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
...
         ClusterDef("4sg_4ac_3cbs_4lgs", num_sgs=4, num_acs=4, num_cbs=3, num_lgs=4, num_lbs=0),
+        ClusterDef("4sg_4ac_6cbs_4lgs", num_sgs=4, num_acs=4, num_cbs=3, num_lgs=4, num_lbs=0),
```
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py [hunk 4]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
...
         ClusterDef("4sg_8ac_3cbs_4lgs", num_sgs=4, num_acs=8, num_cbs=3, num_lgs=4, num_lbs=0),
         ClusterDef("4sg_8ac_6cbs_4lgs", num_sgs=4, num_acs=8, num_cbs=6, num_lgs=4, num_lbs=0),
         # 8 sync_gateways
         ClusterDef("8sg_4ac_3cbs_8lgs", num_sgs=8, num_acs=4, num_cbs=3, num_lgs=8, num_lbs=0),
+        ClusterDef("8sg_4ac_6cbs_8lgs", num_sgs=8, num_acs=4, num_cbs=3, num_lgs=8, num_lbs=0),
```
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py [hunk 5]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
...
         ClusterDef("8sg_8ac_3cbs_8lgs", num_sgs=8, num_acs=8, num_cbs=3, num_lgs=8, num_lbs=0),
+        ClusterDef("8sg_8ac_6cbs_8lgs", num_sgs=8, num_acs=8, num_cbs=3, num_lgs=8, num_lbs=0),
```
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py (hunk 1 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py [hunk 6]
```diff
@@ -426,15 +426,22 @@ def generate_clusters_from_pool(pool_file):
...
         ClusterDef("8sg_12ac_3cbs_8lgs", num_sgs=8, num_acs=12, num_cbs=3, num_lgs=8, num_lbs=0),
         # 16 sync_gateways
+        ClusterDef("16sg_4ac_3cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
+        ClusterDef("16sg_4ac_6cbs_16lgs", num_sgs=16, num_acs=8, num_cbs=3, num_lgs=16, num_lbs=0),
```
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Should num_cbs be 6?

File: libraries/utilities/generate_clusters_from_pool.py (hunk 2 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

File: libraries/utilities/generate_clusters_from_pool.py (hunk 4 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

File: libraries/utilities/generate_clusters_from_pool.py (hunk 3 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

File: libraries/utilities/generate_clusters_from_pool.py (hunk 5 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

File: libraries/utilities/generate_clusters_from_pool.py (hunk 6 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

File: libraries/utilities/generate_clusters_from_pool.py (hunk 1 above)
Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

Comment (sethrosetter) [inline libraries/utilities/generate_clusters_from_pool.py]: Done

**Pull request merged**